LayoutTests/compositing/filters/backdrop-filter-root-element-backdrop-root.html

```diff
@@ -0,0 +1,25 @@
+<!doctype HTML>
+<html style="background-color:transparent">
+<head>
+  <script type="text/javascript" charset="utf-8">
+    if (window.testRunner) {
+        testRunner.dumpAsText();
+        testRunner.waitUntilDone();
+    }
+
+    function doTest()
+    {
+        if (window.testRunner) {
+            document.getElementById('layers').innerText = window.internals.layerTreeAsText(document);
+            testRunner.notifyDone();
+        }
+    }
+
+    window.addEventListener('load', doTest, false);
+  </script>
+</head>
+<body>
```

Comment:
I find it helpful to have a paragraph describing what to look for in the layer tree. I've accidentally regressed things while rebaselining dumps and looking for the wrong thing.